packages/cloud-cognitive/src/components/AboutModal/AboutModal.js

```diff
@@ -12,8 +12,10 @@ import { useResizeDetector } from 'react-resize-detector';
 // Other standard imports.
 import PropTypes from 'prop-types';
 import cx from 'classnames';
+
 import { pkg } from '../../settings';
 import uuidv4 from '../../global/js/utils/uuidv4';
+import { getDevtoolsProps, getDevtoolsId } from '../../internal/devtools';
```

Comment:
Our common JS helpers have up to now gone into `../../global/js/utils`. `../../internal` matches what Carbon do in their `react` package, so we could certainly start reorganising where our shared/common functions go, but I wasn't sure whether you meant to be suggesting that.

Reply:
Good point! I'll revert to `global/js/utils` and if we decide to reorganize, that can happen independently of this.